### ISSUE TITLE
Custom dark mode color setting breaks completely - I believe this is due to Obsidian removing Translucency in the latest beta.

### ISSUE THREAD

**Issue by DudeThatsErin** [damiankorcz/Prism-Theme#158]
**Describe the bug**
When I enabled the theme and have custom selected, I can't use the theme anymore. I have closed and reopened Obsidian but the theme is unusuable.

**To Reproduce**
1. Enable theme
2. Enable Style Settings
3. Change Theme Option for the color (I think) to Custom

**Expected behavior**
A custom theme be applied and allow me to change it.

**Screenshots**
![image](https://github.com/damiankorcz/Prism-Theme/assets/2681022/b0cf2826-e033-4e7f-914f-aae7d98996a9)
![image](https://github.com/damiankorcz/Prism-Theme/assets/2681022/ec1f00a3-f6d1-42c9-b2d4-b60f5f192bc3)

**Setup Information**
 - Device Type: Desktop
 - Operating System: Windows 11 Home
 - Prism version: Latest version, I checked for updates before applying this theme. I can't see anything in Obsidian right now.
 - Color Schemes Used: Custom
 - Obsidian Version: 1.5.10
 - Obsidian Installer Version: 1.5.8
 - Related Community plugins: 
	Plugins installed: 23
	Plugins enabled: 17
		1: Banners v2.0.5-beta
		2: BRAT v0.8.4
		3: Checklist v2.2.13
		4: DataLoom v8.15.12
		5: Calendar v1.5.10
		6: Dataview v0.5.64
		7: Discord Rich Presence v1.5.1
		8: Just Share Please v0.1.5
		9: Kanban v1.5.3
		10: Omnivore v1.9.4
		11: Paste image rename v1.6.1
		12: Settings Search v1.3.10
		13: Style Settings v1.0.7
		14: Templater v2.2.3
		15: Taskbone v2.3.0
		16: Git v2.24.0
		17: Better Daily Notes Plugin v0.3.3

It is not my plugins or settings because it happens in the sandbox vault as well with only Prism and Style Settings installed.

**Additional context**
Latest release, Obsidian disabled translucency for Windows due to an upstream Electron bug they are fixing and it seems like this "custom" color is trying to add that and breaking due to that not being an option anymore.

**Comment by damiankorcz** [comment]
Hi @DudeThatsErin,
The custom option in Color Schemes is intended to be used when you create your own Color Scheme. More info on that here:
https://github.com/damiankorcz/Prism-Theme/wiki/Creating-a-Custom-Colour-Scheme-CSS-Snippet

As to recovering your Vault back to normal:
1. Turn off Obsidian.
2. Navigate to your vault's main folder.
3. In there go to: `.obsidian/plugins/obsidian-style-settings/data.json`
4. Open the file and edit the following lines to match:
```json
"obsidian-prism-theme@@color-schemes-lt": "pt-color-scheme-swan-lt",
```
```json
"obsidian-prism-theme@@color-schemes-dt": "pt-color-scheme-raven-dt",
```
5. Save the file and reopen your vault.

I'll see if I can make this less confusing in a future update.

**Comment by damiankorcz** [comment]
Hi @DudeThatsErin,
This issue should now be resolved in Prism 3.5.0.
I've made the custom option utilise the colours of the Swan and Raven Colour scheme for Light and Dark respectively; preventing the issue you had. This way you can still overwrite the colours but you don't end up with an unusable vault 😅
